src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContentAccordionSection.tsx

```diff
@@ -226,7 +226,11 @@ const ReleaseContentAccordionSection = ({
               )}
 
               <ButtonGroup className="govuk-!-margin-bottom-8 dfe-justify-content--center">
-                <Button variant="secondary" onClick={addBlock}>
+                <Button
+                  variant="secondary"
+                  onClick={addBlock}
+                  preventDoubleClick={false}
```

Comment:
Is this a more general issue with asynchronous button onClicks, will **all** buttons potentially be announcing "unavailable" as they go disabled for a moment? @amyb-hiveit what do you think about this solution?

It seems the crux of the issue is that the user is left with their focus on a disabled input for a moment. We could manage their focus, or stop buttons with prevented double clicks from disabling the button, instead just turn off the functionality for a short duration.